docs/docsite/rst/conf.py

```diff
@@ -227,7 +229,6 @@
 html_context = {
     'display_github': 'True',
     'show_sphinx': False,
-    'is_eol': False,
```

Comment:
I'm still in the middle of the review, but it appears to me that a lot of what's happening here is over-engineering. I'd rather compute this dynamically and have this context for the main Jinja rendering mechanism to do its thing.